### PR TITLE
Fixes clicks in darkness

### PR DESCRIPTION
**Pull request by Razharas**
Makes darkness perfectly clickable
Fix https://github.com/tgstation/-tg-station/issues/5386
I know commit history looks like im not very good coder but its simply not true so dont pay attention to this

### CODE REVIEW

File: code/__DEFINES/preferences.dm [hunk 1]
```diff
@@ -43,4 +43,4 @@
 #define BE_GANG			4096
 #define BE_SHADOWLING	8192
 #define BE_ABDUCTOR		16384
-#define BE_REVENANT		32768
+#define BE_REVENANT		32768
```
Comment (Cheridan) [inline code/__DEFINES/preferences.dm]: lol

**Comment by duncathan** [comment]
>almost as many commits as files changed

File: code/__DEFINES/preferences.dm (hunk 1 above)
Comment (Razharas) [inline code/__DEFINES/preferences.dm]: Didnt even touch this file
I dunno why git artificially engorges my commits with changes

**Comment by Dorsisdwarf** [comment]
You didn't see the original Self Destruct, huh? 

More commits that *lines* changed

**Comment by Razharas** [comment]
I saw one where some folks though they are pulling form us but in reality made a pull request
And then another one of the same type

File: code/__DEFINES/preferences.dm (hunk 1 above)
Comment (optimumtact) [inline code/__DEFINES/preferences.dm]: some editors don't add explicit lineending on final line

yours probably does

**Pull request merged**